systemtest/main_test.go

```diff
@@ -28,6 +28,13 @@ func TestMain(m *testing.M) {
 	if err := StartStackContainers(); err != nil {
 		log.Fatal(err)
 	}
+	log.Println("INFO: cleaning up Elasticsearch...")
+	if err := cleanupElasticsearch(); err != nil {
+		log.Fatal(err)
+	}
+	if err := InitFleet(); err != nil {
```

Comment:
How come we initialize fleet both here and in line 38?

Reply:
because I have not had enough sleep
thanks for catching that, fixing